rules/CodingStyle/Application/UseImportsAdder.php

```diff
@@ -41,9 +41,10 @@ public function addImportsToStmts(
         array $constantUseImportTypes,
         array $functionUseImportTypes
     ): array {
-        $existingUseImportTypes = $this->usedImportsResolver->resolveForStmts($stmts);
-        $existingConstantUseImports = $this->usedImportsResolver->resolveConstantImportsForStmts($stmts);
-        $existingFunctionUseImports = $this->usedImportsResolver->resolveFunctionImportsForStmts($stmts);
+        $usedImports = $this->usedImportsResolver->resolveForStmts($stmts);
+        $existingUseImportTypes = $usedImports->getUseImports();
+        $existingConstantUseImports = $usedImports->getConstantImports();
+        $existingFunctionUseImports = $usedImports->getFunctionImports();
```

Comment:
instead of iterating all the statements for each import type, we do it in one go

Reply:
Looks good :+1: